<ui version="4.0" >
 <class>LabelWidget</class>
 <widget class="QWidget" name="LabelWidget" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QGroupBox" name="gbAlignment" >
     <property name="title" >
      <string>Alignment</string>
     </property>
     <layout class="QGridLayout" >
      <property name="horizontalSpacing" >
       <number>3</number>
      </property>
      <property name="verticalSpacing" >
       <number>3</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lPosition" >
        <property name="text" >
         <string>Position</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2" >
       <widget class="QComboBox" name="cbPosition" >
        <item>
         <property name="text" >
          <string>left</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>center</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>right</string>
         </property>
        </item>
        <item>
         <property name="text" >
          <string>custom</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="0" column="3" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4" >
       <widget class="QLabel" name="lPositionX" >
        <property name="text" >
         <string>x:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5" >
       <widget class="QLineEdit" name="lePositionX" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="6" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType" >
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>16</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="7" >
       <widget class="QLabel" name="lPositionY" >
        <property name="text" >
         <string>y:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="8" >
       <widget class="QLineEdit" name="lePositionY" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Preferred" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lRotation" >
        <property name="text" >
         <string>Rotation</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QLineEdit" name="leRotation" />
      </item>
      <item row="1" column="2" >
       <widget class="QLabel" name="lDegree" >
        <property name="text" >
         <string>Â°</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbBackground" >
     <property name="title" >
      <string>Background</string>
     </property>
     <layout class="QGridLayout" >
      <property name="horizontalSpacing" >
       <number>3</number>
      </property>
      <property name="verticalSpacing" >
       <number>3</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lFilling" >
        <property name="text" >
         <string>Filling</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>21</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2" >
       <widget class="QRadioButton" name="rbFilling0" >
        <property name="text" >
         <string>transparent</string>
        </property>
        <property name="checked" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="3" >
       <widget class="QRadioButton" name="rbFilling1" >
        <property name="text" >
         <string>colour</string>
        </property>
       </widget>
      </item>
      <item row="0" column="4" >
       <widget class="QPushButton" name="bFillingColor" >
        <property name="enabled" >
         <bool>false</bool>
        </property>
        <property name="sizePolicy" >
         <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="toolTip" >
         <string>click to select a colour</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QCheckBox" name="chbBox" >
        <property name="text" >
         <string>Bounding box</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="chbShadow" >
        <property name="text" >
         <string>Shadow</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="gbText" >
     <property name="title" >
      <string>Text</string>
     </property>
     <layout class="QGridLayout" >
      <property name="horizontalSpacing" >
       <number>5</number>
      </property>
      <property name="verticalSpacing" >
       <number>5</number>
      </property>
      <item row="0" column="0" >
       <widget class="QLabel" name="lFont" >
        <property name="text" >
         <string>Font</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" >
       <widget class="QLabel" name="lFontColor" >
        <property name="text" >
         <string>Colour</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0" >
       <widget class="QCheckBox" name="chbUseTex" >
        <property name="text" >
         <string>Tex-syntax</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" >
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="2" colspan="2" >
       <widget class="QFrame" name="frameFontOptions" >
        <property name="frameShape" >
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="spacing" >
          <number>0</number>
         </property>
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="bFontBold" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text" >
            <string>B</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bFontItalic" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <italic>true</italic>
            </font>
           </property>
           <property name="text" >
            <string>I</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bFontUnderline" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="font" >
            <font>
             <underline>true</underline>
            </font>
           </property>
           <property name="text" >
            <string>U</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>15</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="bFontSuperscript" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text" >
            <string>xÂ²</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bFontSubscript" >
           <property name="maximumSize" >
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="text" >
            <string>x_2</string>
           </property>
           <property name="checkable" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer>
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType" >
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" >
            <size>
             <width>15</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QComboBox" name="cbSmallGreekLetters" >
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbBigGreekLetters" >
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cbSymbolLetters" >
           <property name="maximumSize" >
            <size>
             <width>70</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0" colspan="4" >
       <widget class="QTextEdit" name="teLabel" />
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
